feat(python): Add keep-alive option (#9498)
getsentry/sentry-docs: docs/platforms/python/crons/troubleshooting.mdx

Currently, we only support crontab expressions with five fields.

Still having trouble? Reach out to [crons-feedback@sentry.io](mailto:crons-feedback@sentry.io).

### Why Are My Monitors Showing Up as Failed?

> The SDK might be experiencing network issues. See <PlatformLink to="/troubleshooting/#network-issues">this section on troubleshooting connection problems</PlatformLink>.

Reviewer: ```suggestion
The SDK might be experiencing network issues. Learn more about <PlatformLink to="/troubleshooting/#network-issues">troubleshooting network issues</PlatformLink>.
```
